mmv1/third_party/terraform/tests/resource_container_cluster_test.go.erb

```diff
@@ -2502,6 +2502,45 @@ func TestAccContainerCluster_errorNoClusterCreated(t *testing.T) {
 	})
 }
 
+func TestAccContainerCluster_withExternalIpsConfig(t *testing.T) {
+	t.Parallel()
+
+	clusterName := fmt.Sprintf("tf-test-cluster-%s", randString(t, 10))
+	pid := getTestProjectFromEnv()
+
+	vcrTest(t, resource.TestCase{
+		PreCheck:     func() { testAccPreCheck(t) },
+		Providers:    testAccProviders,
+		CheckDestroy: testAccCheckContainerClusterDestroyProducer(t),
+		Steps: []resource.TestStep{
+			{
+				Config: testAccContainerCluster_withExternalIpsConfigEnabled(pid, clusterName),
+			},
+			{
+				ResourceName:            "google_container_cluster.with_external_ips_config",
+				ImportState:             true,
+				ImportStateVerify:       true,
+			},
```

Comment:
This step seems redundant? you already have `testAccContainerCluster_withExternalIpsConfig` with a value of `true`

Reply:
Update applied